CHANGES.md

```diff
@@ -85,6 +86,8 @@ when running on 3.8. ([#31192](https://github.com/apache/beam/issues/31192))
 ## Bugfixes
 
 * (Java) Fixed custom delimiter issues in TextIO ([#32249](https://github.com/apache/beam/issues/32249), [#32251](https://github.com/apache/beam/issues/32251)).
+* (Java, Python, Go) Fixed PeriodicSequence backlog bytes reporting which was preventing Dataflow Runner autoscaling from functioning properly ([#32506](https://github.com/apache/beam/issues/32506)).
+* (Java) Fix improper decoding of schemas with compatible encodingPositions but changed field order with nullable fields ([#32388](https://github.com/apache/beam/issues/32388)).
```

Comment:
don't have context here but wondering if this would sound more clear if accurate:  Fixed improper decoding of schemas, which have compatible encodingPositions but different field order within nullable fields

Reply:
I reworded and think it's clearer now.